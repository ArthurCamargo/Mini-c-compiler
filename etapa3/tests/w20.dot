digraph G {
n0;
n0 -> n1;
n1;
n1 -> n2;
n1 -> n3;
n1 -> n4;
n2;
n2 [label="a"];
n3;
n3 -> n5;
n3 -> n6;
n5;
n5 [label="a"];
n6;
n6 [label="b"];
n3 [label="="];
n4;
n4 -> n7;
n4 -> n8;
n4 -> n9;
n7;
n7 [label="a"];
n8;
n8 [label="3"];
n9;
n9 -> n10;
n9 -> n11;
n10;
n10 [label="b"];
n11;
n11 [label="a"];
n9 [label="="];
n4 [label="="];
n1 [label="if"];
n0 [label="f"];

}