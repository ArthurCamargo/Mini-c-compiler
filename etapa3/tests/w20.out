0x564f8085b350
0x564f8085b350, 0x564f8085b880
0x564f8085b880
0x564f8085b880, 0x564f8085b4c0
0x564f8085b880, 0x564f8085b5f0
0x564f8085b880, 0x564f8085b700
0x564f8085b4c0
0x564f8085b4c0 [label="a"];
0x564f8085b5f0
0x564f8085b5f0, 0x564f8085b530
0x564f8085b5f0, 0x564f8085b5a0
0x564f8085b530
0x564f8085b530 [label="a"];
0x564f8085b5a0
0x564f8085b5a0 [label="b"];
0x564f8085b5f0 [label="="];
0x564f8085b700
0x564f8085b700, 0x564f8085b660
0x564f8085b700, 0x564f8085b6b0
0x564f8085b700, 0x564f8085b830
0x564f8085b660
0x564f8085b660 [label="a"];
0x564f8085b6b0
0x564f8085b6b0 [label="3"];
0x564f8085b830
0x564f8085b830, 0x564f8085b770
0x564f8085b830, 0x564f8085b7e0
0x564f8085b770
0x564f8085b770 [label="b"];
0x564f8085b7e0
0x564f8085b7e0 [label="a"];
0x564f8085b830 [label="="];
0x564f8085b700 [label="="];
0x564f8085b880 [label="if"];
0x564f8085b350 [label="f"];
